packages/frontend/src/view/components/tables/OffersTable.tsx

```diff
@@ -87,11 +93,11 @@ function toStatusType(status: OfferEntry['status']): StatusType {
 function toStatusText(status: OfferEntry['status']): string {
   switch (status) {
     case 'CREATED':
-      return 'CREATED (1/5)'
+      return 'CREATED (1/3)'
```

Comment:
I think previous labels were right. Here are all the possible statuses of force trade offers:
- CREATED (1/5)
- ACCEPTED (2/5)
- SENT (3/5)
- MINED (4/5) = visible on Ethereum blockchain
- INCLUDED (5/5) = included in state update

Reply:
This relates to my previous comment - are we correctly displaying the last two statuses in offerts table on user page?

Reply:
Oh, there are two more statuses that the trade offer can have:

- CANCELLED
- EXPIRED